<design> index.html 추가       >>

hihibara opened the pull request:
### 1. <design> index.html
- 기본적인 .css와 images는 추가했기에 이제 index.html을 upload합니다. (index.html은 템플릿을 하나 따왔습니다.)

### 2.<feat> Index.html에서 LogOut 기능추가 & 로그인 성공하면 닉네임 보여주기

- Login.html에서 로그인을 성공하면 CallBack페이지에서 user 정보를 cookie로 저장하게 되고 그것을 index.html에서 닉네임이 login버튼이 있던 곳에 생기며 Login 버튼은 LogOut버튼으로 변경된다. .

hihibara committed:
<design> index.html 추가       >>
>> 기본적인 .css와 images는 추가했기에 이제 index.html을 upload합니다
>>

Enoch-Kim commented:
@hihibara PR 설명 더 자세히 적는게 어떨까요? 그리고 전달하는 대상을 특정하지 않고 팀 모두에게 공유될 수 있도록 작성하면 좋을 것 같습니다.

hihibara committed:
<doc>    <script>태그에서 constant.js src 제거

굳이 있을 필요가 login.html에서 추가를 할 예정이므로 중복으로 사용되면 오류가 발생할 수 있기에 index,html에서 제거

hihibara committed:
<feat> Index.html에서 LogOut 기능추가 & 로그인 성공하면 닉네임 보여주기

Login.html에서 로그인을 성공하면 CallBack페이지에서 user 정보를 cookie로 저장하게 되고 그것을 index.html에서 닉네임이 login버튼이 있던 곳에 생기며
Login 버튼은 LogOut버튼으로 변경된다. .
